configuration/persistence.md

```diff
@@ -63,13 +67,16 @@ The syntax is as follows:
 
 ```java
 Items {
-    <itemlist1 [-> "<alias1>"] : [strategy = <strategy1>, <strategy2>, ...]
+    <itemlist1> [-> "<alias1>"] : [strategy = <strategy1>, <strategy2>, ...]
     <itemlist2> [-> "<alias2>"] : [strategy = <strategyX>, <strategyY>, ...]
     ...
 
 }
 ```
 
+Note that you may omit the `strategy` portion of the `itemlist` so long as you have provided a `default` strategy in the `Strategies` section.
```

Comment:
`so long as` => `as long as`

Reply:
Yeah - 'so long as' and 'as long as' are superfine distinctions in English.  'as long as' is much more common, so this is what I should use in a document being read by people from all over the world.  In any case, I am proposing to replace all of this anyway, but thank you for pointing this out.

Reply:
Ahh - from dictionary.cambridge.org (what do they know about English??!!)
As long as: refer to the intended duration of a plan or idea
As long as and so long as: means 'provided that", 'providing that" or 'on condition that'.  So long as is a little informal.

Reply:
To be honest i didn't think of any rules. It sounded strange to me while reading. 😄 
That was my intention to comment it here.

Reply:
I agree.  Well, I will rewrite the section and drop this strange wording anyway.